app/i18n/fr/gen.php

```diff
@@ -204,6 +204,7 @@
 	),
 	'share' => array(
 		'Known' => 'Sites basés sur Known',
+		'archiveIS' => 'archive.is',	// TODO
```

Comment:
```suggestion
		'archiveIS' => 'archive.is',	// IGNORE
```

And for (most or all) other languages as well

Reply:
I forgot it 🙈